src/coreclr/ToolBox/superpmi/superpmi/jitinstance.cpp

```diff
@@ -322,12 +322,12 @@ JitInstance::Result JitInstance::CompileMethod(MethodContext* MethodToCompile, i
         {
             // For altjit, treat SKIPPED as OK
 #ifdef TARGET_AMD64
```

Comment:
I wonder if these TARGET ifdefs should be generalized, e.g.
```
// If the target architecture doesn't match the expected target architecture, then we've got an altjit, so treat SKIPPED as OK to avoid counting the compilation as failed.
DWORD expectedTargetArchitecture = pParam->pThis->mc->repGetExpectedTargetArchitecture();
if (((GetSpmiTargetArchitecture() == SPMI_TARGET_ARCHITECTURE_AMD64) && (expectedTargetArchitecture != IMAGE_FILE_MACHINE_AMD64)) ||
if (((GetSpmiTargetArchitecture() == SPMI_TARGET_ARCHITECTURE_X86) && (expectedTargetArchitecture != IMAGE_FILE_MACHINE_I386))
if (((GetSpmiTargetArchitecture() == SPMI_TARGET_ARCHITECTURE_ARM) && (expectedTargetArchitecture != IMAGE_FILE_MACHINE_ARMNT))
if (((GetSpmiTargetArchitecture() == SPMI_TARGET_ARCHITECTURE_ARM64) && (expectedTargetArchitecture != IMAGE_FILE_MACHINE_ARM64)))
{
        jitResult = CORJIT_OK;
}
```